src/openrct2/object/SmallSceneryObject.cpp

```diff
@@ -193,34 +193,34 @@ void SmallSceneryObject::PerformFixes()
         identifier == "TTRFWD07" ||
         identifier == "TTRFWD08")
     {
-        static const rct_object_entry scgMine = GetScgMineHeader();
-        SetPrimarySceneryGroup(&scgMine);
+        static const auto& scgMine = GetScgMineHeader();
+        SetPrimarySceneryGroup(scgMine);
     }
 
     // ToonTowner's glass roofs. Make them show up in the Abstract Theming.
     if (identifier == "TTRFGL01" ||
         identifier == "TTRFGL02" ||
         identifier == "TTRFGL03")
     {
-        static const rct_object_entry scgAbstr = GetScgAbstrHeader();
-        SetPrimarySceneryGroup(&scgAbstr);
+        static const auto& scgAbstr = GetScgAbstrHeader();
+        SetPrimarySceneryGroup(scgAbstr);
     }
 }
 // clang-format on
 
-rct_object_entry SmallSceneryObject::GetScgPiratHeader()
+ObjectEntryDescriptor SmallSceneryObject::GetScgPiratHeader()
 {
-    return Object::CreateHeader("SCGPIRAT", 169381767, 132382977);
+    return ObjectEntryDescriptor("rct2.scgpirat");
 }
 
-rct_object_entry SmallSceneryObject::GetScgMineHeader()
+ObjectEntryDescriptor SmallSceneryObject::GetScgMineHeader()
 {
-    return Object::CreateHeader("SCGMINE ", 207140231, 3638141733);
+    return ObjectEntryDescriptor("rct2.scgpirat");
 }
 
-rct_object_entry SmallSceneryObject::GetScgAbstrHeader()
+ObjectEntryDescriptor SmallSceneryObject::GetScgAbstrHeader()
 {
-    return Object::CreateHeader("SCGABSTR", 207140231, 932253451);
+    return ObjectEntryDescriptor("rct2.scgabstr");
```

Comment:
These getters should probably be `const` functions

(Dunno why but GitHub didn't allow me to suggest changes on each individually)